### PR TITLE
Move 4.10 to build image from repository

### PR DESCRIPTION
**Pull request by perdasilva**
Signed-off-by: perdasilva <perdasilva@redhat.com>

### CODE REVIEW

**Comment by perdasilva** [comment]
Failures are expected here since the .ci-operator.yaml hasn't been updated yet

File: ci-operator/config/openshift/operator-framework-olm/openshift-operator-framework-olm-release-4.10.yaml [hunk 1]
```diff
@@ -26,7 +24,13 @@ images:
       as:
       - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.10
   to: operator-registry
+- dockerfile_path: base.Dockerfile
+  from: src
+  to: ci-image
 promotion:
+  disabled: true
+  excluded_images:
+  - ci-image
```
Comment (bparees) [inline ci-operator/config/openshift/operator-framework-olm/openshift-operator-framework-olm-release-4.10.yaml:33]: why are you disabling promotion generally?  i think you just want to exclude ci-image.

Comment (perdasilva) [reply ci-operator/config/openshift/operator-framework-olm/openshift-operator-framework-olm-release-4.10.yaml:32]: Great catch. I took the definition from 4.11. The release guys must have added it for their purposes. I've removed it!

**Pull request merged**